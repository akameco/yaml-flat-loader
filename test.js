import test from "ava";
import m from ".";

test("snapshot yaml", t => {
  const input = `
---
hello: world
  `;
  t.snapshot(m(input));
});

test("snapshot nest yaml", t => {
  const input = `
---
welcome:
  to:
    yaml
  `;
  t.snapshot(m(input));
});
