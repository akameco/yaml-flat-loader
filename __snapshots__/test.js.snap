// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot nest yaml 1`] = `
"{
	\\"welcome.to\\": \\"yaml\\"
}"
`;

exports[`snapshot yaml 1`] = `
"{
	\\"hello\\": \\"world\\"
}"
`;
